pom.xml

```diff
@@ -124,6 +127,7 @@
     <yarn.version>${hadoop.version}</yarn.version>
     <zookeeper.version>3.4.6</zookeeper.version>
     <curator.version>2.7.1</curator.version>
+    <okapi.version>0.4.2</okapi.version>
```

Comment:
Is this published? I don't see it in Maven Central. Maybe that's the problem with building? Shouldn't this wait for 1.0.0?

Reply:
That's not the build issue. We published an updated version to Maven Central yesterday so perhaps the search cache was not yet updated when you looked. It is definitely there: https://repo1.maven.org/maven2/org/opencypher/okapi-shade/0.4.2/

We're building this JAR specifically for this project. Since this is still in review we're continuously improving it based on review feedback and thus we must bump versions in between. The plan is to move this to 1.0.0 once the review process is completed or nearing its end, so that we can set in stone the exact shape of the dependency in an acceptable state.

Reply:
Sounds good, yes I see it now. That wasn't the issue.